sktime/forecasting/model_selection/_tune.py

```diff
@@ -483,9 +485,33 @@ def __init__(
         )
         self.param_grid = param_grid
 
+    def _check_param_grid(self, param_grid):
+        """_check_param_grid from sklearn 1.0.2, before it was removed."""
+        if hasattr(param_grid, "items"):
```

Comment:
seems like an indirect way to check that p is a dictionary.

Reply:
yes, I know. This is 1:1 copied from sklearn. A proper refactor would probably just ditch the function entirely and replace it by better check functionality. Perhaps that´s what happened in 1.1.0, but I don´t think the time investment is worth it, separate from a proper clean-up.